lib/constants.js

```diff
@@ -446,6 +446,14 @@ exports.PREFERRED_CONTACT_METHODS = [
 ]
 
 exports.PRODUCT_CATAGORIES = [
+  {
+    value: 'grains',
+    label: 'Grains'
+  },
+  {
+    value: 'other_row_crops',
+    label: 'Other row crops'
+  },
   {
     value: 'grains_other',
```

Comment:
so this doesnt need to appear in the schema?

Reply:
Notionally no, but leaving it on makes this backwards compatible with existing farm groups.